app/channels/chat_channel.rb

```diff
@@ -0,0 +1,5 @@
+class ChatChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "chat_#{params[:chat_id]}"
```

Comment:
you can just `stream_for chat` where you can find chat with `Chat.find(params[:chat_id])`
Anyway this is ok, it's just a comment as I think that in this way is easier